.travis.yml

```diff
@@ -23,7 +23,7 @@ install:
 before_script:
   - pwd
   - wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.3/swagger-codegen-cli-2.2.3.jar -O swagger-codegen-cli.jar
-  - java -jar swagger-codegen-cli.jar generate -i https://dockstore.org:8443/swagger.json -l typescript-angular2 -o src/app/shared/swagger
+  - java -jar swagger-codegen-cli.jar generate -i https://raw.githubusercontent.com/ga4gh/dockstore/develop/dockstore-webservice/src/main/resources/swagger.yaml -l typescript-angular2 -o src/app/shared/swagger
```

Comment:
you're probably going to want a tagged version or at least a specific SHA1 commit unless you want this to break an inconvenient times